packages/graph-explorer/src/core/ConnectorProvider/ConnectorProvider.tsx

```diff
@@ -74,9 +74,9 @@ const ConnectorProvider = ({ children }: PropsWithChildren<any>) => {
     // connector instance is only rebuilt if any connection attribute change
     if (!isSameConnection(prevConnection, config?.connection)) {
       setConnector({
-        explorer: getExplorer(
-          (config?.connection as ConnectionConfig) || undefined
-        ),
+        explorer: config?.connection
```

Comment:
The logic has changed here, is that correct?

Reply:
Good eye. I wrestled with that one a bit. What do you think it should be?

**Before**
If `config` or `config.connection` is `undefined` then we would get `gremlinExplorer`

**After**
If `config` or `config.connection` is `undefined` then we would get `undefined`

Reply:
I think we should get default `gremlinExplorer`

Reply:
ok cool. I'll make that change.